### ISSUE TITLE
ShearDisplacementStructure vs DiscreteDiscontinuity

### ISSUE THREAD

**Issue by mbeaufils** [opengeospatial/Geotech#25]
Do we need both? Is there a relationship between them?

#20 
#23

**Comment by dponti** [comment]
My impression is that a shear displacement structure is a specialization of a discrete discontinuity - eg. a discontinuity as a result of displacement across the interface. Or is the intent to distinguish scale - eg. a fault zone with significant extent as opposed to small-scale fractures, gouge, etc. that could be observed either within ore outside of a fault zone?

**Comment by teemulind** [comment]
From my point of view ShearDisplacementStructure indicates a geologic structure in the ground where something has moved (sheared, slipped, deformed). I.e. how the structure has come to existence in the first place. When it comes to Discrete discontinuity the point of view is the continuity (or discontinuity) of the feature of interest. That is fundamentally different thing than movement. From my point of view these should not be mixed.

**Comment by Didymograptus** [comment]
On a broader note, can we agree on a set of terms defined by an established publication. For example (only) a dictionary of geology https://geology.com/dictionary/glossary-d.shtml Let's pick one publication and stick with it as far as possible.  It will save a lot of discussion.

**Comment by Alexis-SERIEYS** [comment]
From my point of view, this two objects came from two different approach :

- ShearDisplacementStructure : This concept came from a gelogical approach, in order to describe the specific case of a discontinuity marked by a Shear displacement,

- DiscreteDiscontinuity : This concept came from a geotechnical point of view, in order to affect a geometry and mechanical properties at one particular discontinuity. In this concept, the discontinuity is not necessarily a "tectonic structure", but could be any discontinuity used in the design model. 

So from my point of view, i think these are two different concept. To describe geotechnical project, i think at least the object "Discrete Discontinuity" Should be conserved this way.

**Comment by teemulind** [comment]
In geology, movement (displacement) is not the only point of view. For instance, geology also has the rheological point of view which coud perhaps suit better for engineering purposes. In the case of geological structures, most typical classes are brittle structures and ductile structures. Brittle structures are sharp breaks in geological media. And a break is a certain type of discontinuity. I think it is not necessary to include the displacement point of view if it is not needed.

Considering what is a discontinuity, I would recommend the following research report published by British Geological survey.
http://nora.nerc.ac.uk/id/eprint/13986/1/RR10005.pdf

**Comment by Alexis-SERIEYS** [comment]
@teemulind 
I agree with you on the "displacement" point. So maybe, it is relevant to rename this object which is par of the "Domain-Geology".

But I wanted to say that it was important to keep a specific object in the "Domain Geotechnic" Such as "Discrete discontinuity" in order to represent the hypothesis of design :

- Mechanical parameters : (c', phi')
- Position of the discontinuity, in geotechnics this could be an hypothesis : for exemple in the case of an excavation wich this discontinuity could not observed directly on the terrain, but the discontinuity set of the rock mass is knowned and allow such discontinuity at this psecific position.

**Comment by teemulind** [comment]
I wonder what is our aim in this discussion(?). Is it to choose an appropriate terminology for "natural structures in the earth" for engineering purposes? If yes, we should also consider the standardisation in general. The more we have different terms for similar kind of features in the ground the more difficult it will be to enable data flow in future. For this reason, I think it is important to consider what these terms/concepts really mean and then choose an appropriate one.

**Comment by mbeaufils** [comment]
The ShearDisplacementStructure is already defined by CGI IUGS in OGC GeoSciML.
DiscreteDiscontinuity is up to us. Both MINnD project and IFC Tunnel GeoSubgroup identified it as a suitable and distinct concept to have.

The aim of this discussion is:
1 - Confirm the pertinence of ShearDisplacementStructure for the geological modeling phase,
2 - Confirm the need for a distinct term called DiscreteDiscontinuity for the geotechnical modeling phase,
3- Identify possible relationship between ShearDisplacementStructure and DiscreteContinuity.

Seems 1 and 2 are confirmed. Now regarding 3:
In GeoSciML, the ShearDisplacementStructure belongs to the family of the GeologyStructure which is defined as "_a configuration of matter in the Earth based on describable inhomogeneity, pattern, or fracture in an earth material.  The identity of a GeologicStructure is independent of the material that is the substrate for the structure.  The general GeologicFeatureRelation (available in the Extension package) is used to associate penetrative GeologicStructures with GeologicUnits.  GeoSciML Basic only provides a limited set of core structures (Contact, Fold, ShearDisplacementStructure and Foliation) with a single property to categorise them.  Supplemental properties and geologic structure types are available from the Extension package_."

Would it make sense to propose DiscreteDiscontinuity as a GeologyStructure?

**Comment by Alexis-SERIEYS** [comment]
@mbeaufils, thanks for this clarification. 

To answer your final question on the point 3, as a geotechnical engineer, I would say that is important to keep a geotechnical object distinct from the geological model. 

Because we could use often discontinuities that couldnot be directly been observed on the field, but considered after an analysis of the discontinuity network of the rock mass and the caracteristics of the ground works. Its mostly the case when you work on the design of an excavation or a tunnel whitin a rock mass.

For exemple, for the design study of an excavation with a rock slope, it is possible to analyse the stability of the slope considering the biggest wedge that could be formed by : 
- The discontinuity network of the rock mass ;
- The shape and dimensions of the excavation.

This wedge is totaly, but need to be analysed during the design studies. As an hypothesis used for the design, it possibly needs a representation in the BIM model, at least in the early phases of the project, even if it not came from an effective field observation.

Sorry for beeing maybe too long, and I hope im not off the topic. But to sum it up, to me its important to keep two independant concepts : one for geologic purpose, and an other geotechnic purpose. 
Possibly, this geotechnical object could, but doesn't need to, be linked with the geological object, in order to refer to certain properties from geological observation.

**Comment by mbeaufils** [comment]
@Alexis-SERIEYS : by defining a relationship between ShearDisplacementStructure and DiscreteContinuity I mean in terms of semantics.

In other words, can we say a DiscreteContinuity is a kind of GeologyStructure (a configuration of matter in the Earth based on describable inhomogeneity, pattern, or fracture in an earth material)?

The need or opportunity to connect a specific DiscreteDiscontinuity to another specific object (may it be a geotechnicalUnit, a geologicUnit, a ShearDisplacementStructure, etc... ) is another topic.

**Comment by Didymograptus** [comment]
Can we park discussion about "ShearDisplacementStructure" because we are really getting into structural geology (our focus is geotech) and put "Discontinuity" at the high level? 

How 'discrete' a "DiscreteContinuity" is depends on the use-case and the point of view of the model. As you zoom into the model you may decide that a 'discrete' fault turns into an 'indiscrete' fault zone. i.e. the nature of the discontinuity should be contained in metadata appropriate to the model and use case.

If we just use "Discontinuity" we can hang a lot of things off of it that will help to inform the geotechnical interpretation  e.g. is it gradational, is it an unconformity, is there displacement etc. what is the scale of the displacement, is it normal or reverse etc.

**Comment by teemulind** [comment]
The discontinuity as a concept is frequently used in engineering geology/geotechnical engineering related literature. When it comes to the shearDisplacementStructure I agree that it is different concept from discontinuity. Not because of geotech vs structural geology scientific disciples but because discontinuity indicatates continuity/discontinuity of matter – whereas shearDisplacementStructure indicates displacement of matter. However, they are both ”natural structures in the ground”. So in that sense they are both geologic structures. The semantics matter.

When it comes to the scientific disciplines, I wouldn't differentiate geotech strictly from structural geology or some other related discipline such as rock mechanics. They have major overlap in terminology. For instance, the concept of discontinuity is also used in geology depending of the point of view. This can be seen easily by looking some of the most recent papers published from these disciplines.

Furthermore, I wonder what we should do with the terms ’fracture’ and ’joint’ or is that the topic of another discussion? If you look at the link I posted earlier in this discussion a certain type of 'discontinuity' is a synonym for a 'fracture'.

With the aim of building the first conceptul model, I think it all comes down to how we want to call the ’natural structures in the ground’. Shall we use geological terminology which offers more details if needed or shall we generalise them all to ’discontinuity’ and then include the geological information as attributes when needed.
